cfg/qt.cfg

```diff
@@ -5352,7 +5352,7 @@
   <define name="Q_INTERFACES(X)" value=""/>
   <define name="Q_LIKELY(expr)" value="expr"/>
   <define name="Q_NAMESPACE" value=""/>
-  <define name="Q_NULLPTR" value="NULL"/>
```

Comment:
Why was this changed?

Reply:
It should have been removed completely, right?

Reply:
I would say leave it as it was, since we want to handle `Q_NULLPTR` in code that still uses it. The changes in the tests in `qt.cpp` should also be reverted. But then there are pretty much no functional changes left.

Reply:
Ah, I see, you want to test these old macros. Then it is better to leave everything as it is.